### DISCUSSION TITLE
Фильтр по шаблону

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#356] [Как это сделать?] [answered]
Есть возможность использовать шаблоны для фильтров в полях? Например, использование спецсимвола %, как это сделано в MS SQL. % - спецсимвол заменяющий любой набор неизвестных символов.

**Comment by vldekov** [comment]
Возможно использовать ПРОБЕЛ?

![image](https://user-images.githubusercontent.com/114401808/234838371-370fef0d-d335-4cb4-8360-d26a7383b56b.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
https://ru.docs.totum.online/user-guide#filter

### Режимы поиска значений

По умолчанию поиск в фильтрах осуществляется по пересечению совпадений частей введенных через пробел:

`~~:` — так обозначен режим по умолчанию и эту комбинацию символов можно не вводить перед строкой поиска:

Азот (20 счет) найдется если ввести `Аз 20`.

`!~~:` или `!:` — режим по умолчанию с исключением:

Если ввести `!: Аз 20` то покажет все, кроме Азот (20 счет) и аналогичных.

`~:` — пробел как символ:

Азот (20 счет) найдется если ввести `~: от (20`.

`!~:` — исключить + пробел как символ.

`=:` полное соответсвие всего значения:

Азот (20 счет) найдется если ввести `=: Азот (20 счет)`.

`!=:` — инвертированное полное соответствие.

`^:` — значение начинается с значения поиска:

Азот (20 счет) найдется если ввести `^: Азот` или `^: Азот 20`.

`!^:` — инвертированное.

`^~:` — значение начинается + пробел как символ:

Азот (20 счет) найдется если ввести `^: Азот (20`.

`!^~:` — инвертированное.
